website/jsdocs/jsdocs.js

```diff
@@ -131,6 +131,16 @@ function getDocBlock(node, commentsForFile, linesForFile) {
   }
   var docblock;
   var prevLine = node.loc.start.line - 1;
+  var startText = linesForFile[prevLine].trim();
+
+  // Get to actual start of function declaration (for multi-line declarations)
```

Comment:
Good catch @pjjanak! However I don't think this will work; for example there are Flow typehints that contain left parens (namely for function types).

I did some digging, and actually this is only an issue for multi-line class method declarations. It looks like Esprima handles ES6 class methods by creating a `MethodDefinition` AST node which contains another AST node with the actual `FunctionDeclaration` - the `MethodDefinition` has the correct line range.

A simpler fix would be to just call `getDocBlock` on the `MethodDefinition` AST node in `getClassData` near line 385 below.